src/Compat/CompatForm.php

```diff
@@ -3,15 +3,38 @@
 namespace ipl\Web\Compat;
 
 use ipl\Html\Form;
+use ipl\Html\HtmlDocument;
+use ipl\Html\HtmlString;
 use ipl\I18n\Translation;
+use ipl\Web\Common\DuplicateSubmitButton;
 use ipl\Web\FormDecorator\IcingaFormDecorator;
 
 class CompatForm extends Form
 {
     use Translation;
+    use DuplicateSubmitButton;
 
     protected $defaultAttributes = ['class' => 'icinga-form icinga-controls'];
 
+    /**
+     * A duplicate of the Primary Submit Button is being prepended, if there are more than one present.
```

Comment:
You're overriding the base documentation block. Make sure to include the short message here again.